src/ontology/components/mappings_xrefs.owl

```diff
@@ -3890,9 +3890,6 @@
     <rdf:Description rdf:about="http://purl.obolibrary.org/obo/FBbt_00110396">
         <oboInOwl:hasDbXref>FlyBrain_NDB:10613</oboInOwl:hasDbXref>
     </rdf:Description>
-    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/FBbt_00110399">
-        <oboInOwl:hasDbXref>FlyBrain_NDB:10614</oboInOwl:hasDbXref>
```

Comment:
The `mappings_xrefs.owl` is automatically generated from the SSSOM mapping sets (in `src/mappings`).

To remove this xref, you must edit the `src/mappings/flybrain.sssom.tsv` file [here](https://github.com/FlyBase/drosophila-anatomy-developmental-ontology/blob/427c17640794347978ae88066312968dc38c339e/src/mappings/flybrain.sssom.tsv#L513):

```
FBbt:00110399   adult VPNp4 lineage clone       skos:exactMatch FlyBrain_NDB:10614      semapv:ManualMappingCuration
```

Reply:
see https://github.com/FlyBase/drosophila-anatomy-developmental-ontology/pull/1927/commits/b444be91846ac5161e8ee81d153053204bbb6160#diff-e84f5b55a2050e690092cfc5fda1abbcd5dfc9e36b0e952ad3da8d81e83a0440

was just updating the owl in case it was the reason the checks failed (I think more likely to be due to missing labels, but doesn't hurt)

Reply:
Oh, my bad, I was looking only at the last commit, so I missed that you had already edited the SSSOM file. Sorry about that.